### PR TITLE
resolve --log-level deprecation and replace to --verbosity 

### PR DESCRIPTION
**Pull request by cpanato**
- resolve --log-level deprecation and replace to --verbosity 
- upgrade kind to v0.14.0

### CODE REVIEW

**Review by davidkarlsen — APPROVED** [review]
lgtm

File: action.yml [hunk 1]
```diff
@@ -23,15 +23,17 @@ inputs:
...
   kubectl_version:
-    description: "The kubectl version to use (default: v1.21.10)"
+    description: "The kubectl version to use (default: v1.22.10)"
     required: false
-    default: "v1.21.10"
+    default: "v1.22.10"
   install_only:
     description: "Skips cluster creation, only install kind (default: false)"
+    required: false
 runs:
```
[the remark this answers is not in the document]
Comment (cpanato) [reply action.yml:37]: done!

**Comment by cpanato** [comment]
@davidkarlsen updated the node to 16, PTAL

**Review by davidkarlsen — APPROVED** [review]
lgtm

**Pull request merged**